src/www/containers/contactpage/ContactPage.jsx

```diff
@@ -0,0 +1,133 @@
+import "./ContactPage.scss"
+
+import React, { Component } from "react";
+import { clearErrors, contact } from "../../actions/contact";
+
+import Input from "../../components/input/Input";
+import { connect } from "react-redux";
+import { withTranslation } from "react-i18next";
+
+const top_img = '/img/top_img.png'; //Photo by Patrick Perkins on Unsplash
+const form_img = '/img/form_img.png'; //Photo by LinkedIn Sales Navigator on Unsplash
+
+const mapState = state => ({
+  undefined
+});
+
+const mapDispatch = {
+  contact,
+  clearErrors
+};
+
+export default connect(
+  mapState,
+  mapDispatch
+)( withTranslation()(
+class ContactPage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      first_name: '',
+      last_name: '',
+      email: '',
+      question: '',
+      first_nameHighlight: false,
+      last_nameHighlight: false,
+      emailHighlight: false
+    }
+  }
+
+  onChangeQuestion = (e) => {
+    this.setState({ question: e.target.value.trim() })
+  }
+
+  clearHighlights = async () => {
+    this.setState({ first_nameHighlight: false,
+      last_nameHighlight: false,
+      emailHighlight: false,});
+  };
+
+  componentDidMount() {
+    this.props.clearErrors();
+  }
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    this.props.clearErrors();
+    const data = new FormData(event.target);
+
+    const { first_name, last_name, email, question } = this.state;
+
+    if (first_name !== "" && last_name !== "" && email !== "" && question !== "") {
+      this.clearHighlights();
+      this.props.contact(first_name, last_name, email, question);
+      console.log(stringifyFormData(data));
+    } else {
+      console.log("ELSE")
+      this.setState({
+        first_nameHighlight: first_name === "",
+        last_nameHighlight: last_name === "",
+        emailHighlight: email === "",
+      });
+    }
+
+    if(!alert('Thank you!')){window.location.reload();}
+  }
+
+    render() {
+      const { t } = this.props;
+      return (
+        <div className="container">    
+            <img className="top-img" alt="top_img" src={top_img}/>
+            <span className="contact-form-title1">{t("contactpage.title1")}</span>
+            <span className="contact-form-title2">{t("contactpage.title2")}</span>
+            <div className="contact-form-block">
+                <img className="form-image" alt="form_img" src={form_img}/>
+                <form onSubmit={this.handleSubmit} >
```

Comment:
I think you missed the class attribute here, even though it's present in your CSS ruleset.

```suggestion
                <form className="contact-form" onSubmit={this.handleSubmit} >
```